<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackersChanger</class>
 <widget class="QDialog" name="TrackersChanger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Change trackers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="Trackers_">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Info hash</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>URL</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tier</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Next announce</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fails</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Verified</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Updating</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start sent</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Complete sent</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sources</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ButtonAdd_">
       <property name="text">
        <string>Add...</string>
       </property>
       <property name="icon">
        <iconset theme="list-add"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonModify_">
       <property name="text">
        <string>Modify...</string>
       </property>
       <property name="icon">
        <iconset theme="document-edit"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonRemove_">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TrackersChanger</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TrackersChanger</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
